ga4gh/frontend.py

```diff
@@ -66,9 +75,41 @@ def handle_error(exception):
     return response
 
 
```

Comment:
There's a lot of functions here that seem like they should be grouped in a class. What about ```class StatusGenerator(object)``` which has a method `getInfoDict()` or something? We could instantiate this in configure() doing something like
```python
# ....
app.statusGenerator = StatusGenerator()
```
The startupTime can be allocated in the constructor of this. I'd like to avoid module level state where at all possible.